.changeset/config.json

```diff
@@ -11,6 +11,7 @@
   "ignore": [
     "@examples/*",
     "@apps/*",
+    "@apps/browser-extension",
```

Comment:
Why do we need this as we already have `@apps/*`?

Reply:
oh good catch